src/components/ThreadEnvelope.vue

```diff
@@ -197,9 +197,10 @@ export default {
 			}
 		},
 	},
-	mounted() {
+	async mounted() {
 		if (this.expanded) {
-			this.fetchMessage()
+			await this.fetchMessage()
+			this.$nextTick(() => this.scrollTo())
```

Comment:
scroll to *what*? :P

how about `scrollToCurrentEnvelope`?

Reply:
Agreed